### PR TITLE
Added the ec2_run to the policy runner

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Added the ec2_run to the policy runner

### CODE REVIEW

File: cloud_governance/common/utils/utils.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+
+class Utils:
+    pass
```
Comment (ebattat) [inline cloud_governance/common/utils/utils.py:4]: Why we need it ?

File: jenkins/clouds/aws/daily/policies/run_policies.py [hunk 2]
```diff
@@ -91,6 +91,13 @@ def get_policies(type: str = None):
...
                 os.system(f"""podman run --rm --name cloud-governance --net="host" -e EMAIL_ALERT="False" -e account="PERF-SCALE" -e policy="{policy}" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE}" -e AWS_DEFAULT_REGION="{region}" -e dry_run="yes" -e LDAP_HOST_NAME="{LDAP_HOST_NAME}" -e special_user_mails="{special_user_mails}" -e account_admin="{account_admin}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" {es_index_env_var} -e policy_output="s3://{BUCKET_PERF_SCALE}/{LOGS}/{region}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
 
+# Run the EC2 run policy
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/policies/run_policies.py:94]: Why u add a separate code for it ?

**Comment by ebattat** [comment]
@athiruma, fix conflicts

File: cloud_governance/common/utils/utils.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/common/utils/utils.py]: Another PR file, due to PR switching mistakenly added here.  Removed

File: jenkins/clouds/aws/daily/policies/run_policies.py (hunk 2 above)
Comment (athiruma) [reply jenkins/clouds/aws/daily/policies/run_policies.py:94]: In the policy changes, if we pass the env variable **RUN_ACTIVE_REGIONS=True*. It runs the policy in all active regions. Due to that, I don't have a chance to include it in the for loop, for the upcoming PR's statics regions are removed.

Comment (ebattat) [reply jenkins/clouds/aws/daily/policies/run_policies.py:94]: So why you add a whole code ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/policies/run_policies.py:94]: I opened this pr to resolve the ec2-run bug. Later prs i will open new pr to order the policy runs.

**Pull request merged**